source/SpinalHDL/Design errors/width_mismatch.rst

```diff
@@ -5,12 +5,12 @@ Width mismatch
 Introduction
 ------------
 
-SpinalHDL will checks the correctness of signals bitwidth on assignements and on operators inputs.
+SpinalHDL will check that signals on the left and right side of assignments and operators are the same width.
```

Comment:
Shroud `have` be use instead of `are`. this because a width is a propriety someone have (this is my line of reasoning)
I can be wrong with this, so be free to ignore this comment if i'm wrong (valid for all the comments so far)